wikipedia-dark.user.css

```diff
@@ -283,6 +345,32 @@
     background-color: transparent !important;
     border-color: #555 !important;
   }
+  .mw-editfont-monospace {
+    background-color: #222;
+    color: #9a9a9a;
+  }
+  .userSummaryButtons a, #userSummaryButtons a {
+    background-color: #333;
+    border-color: #333;
+  }
+  .userSummaryButtons a:hover, #userSummaryButtons a:hover {
+    background-color: #444;
+    border-color: #444;
+  }
```

Comment:
```css
  .userSummaryButtons a, #userSummaryButtons a {
    background-color: #002800;
    border-color: #061;
    color: #aaa !important;
  }
  .userSummaryButtons a:hover, #userSummaryButtons a:hover {
    background-color: #002800;
    border-color: #061;
    filter: brightness(120%);
  }
```
Please use green colors to replace default green elements, same with other colors/elements and always when possible, sometimes its not possible but this isnt this case.

Reply:
Implemented